diff -Naur pcre-8.39.orig/ltmain.sh pcre-8.39/ltmain.sh
--- pcre-8.39.orig/ltmain.sh	2016-06-14 11:04:39.000000000 +0200
+++ pcre-8.39/ltmain.sh	2016-09-01 22:08:12.936605035 +0200
@@ -8361,7 +8361,7 @@
 	      fi
 	    else
 	      # We cannot seem to hardcode it, guess we'll fake it.
-	      add_dir=-L$libdir
+	      add_dir=-L=$libdir
 	      # Try looking first in the location we're being installed to.
 	      if test -n "$inst_prefix_dir"; then
 		case $libdir in
