diff -Naur iw-4.7.orig/Makefile iw-4.7/Makefile
--- iw-4.7.orig/Makefile	2016-05-31 11:52:46.000000000 +0200
+++ iw-4.7/Makefile	2016-08-19 19:20:09.175712575 +0200
@@ -115,13 +115,13 @@
 	@$(NQ) ' GZIP' $<
 	$(Q)gzip < $< > $@
 
-install: iw iw.8.gz
+install: iw iw.8
 	@$(NQ) ' INST iw'
 	$(Q)$(MKDIR) $(DESTDIR)$(SBINDIR)
 	$(Q)$(INSTALL) -m 755 iw $(DESTDIR)$(SBINDIR)
 	@$(NQ) ' INST iw.8'
 	$(Q)$(MKDIR) $(DESTDIR)$(MANDIR)/man8/
-	$(Q)$(INSTALL) -m 644 iw.8.gz $(DESTDIR)$(MANDIR)/man8/
+	$(Q)$(INSTALL) -m 644 iw.8 $(DESTDIR)$(MANDIR)/man8/
 
 clean:
 	$(Q)rm -f iw *.o *~ *.gz version.c *-stamp
