diff -Naur libnl-3.2.25.orig/build-aux/ltmain.sh libnl-3.2.25/build-aux/ltmain.sh
--- libnl-3.2.25.orig/build-aux/ltmain.sh	2014-07-16 12:02:32.000000000 +0200
+++ libnl-3.2.25/build-aux/ltmain.sh	2015-10-25 18:43:10.741422052 +0100
@@ -6918,7 +6918,7 @@
 	      fi
 	    else
 	      # We cannot seem to hardcode it, guess we'll fake it.
-	      add_dir="-L$libdir"
+	      add_dir="-L=$libdir"
 	      # Try looking first in the location we're being installed to.
 	      if test -n "$inst_prefix_dir"; then
 		case $libdir in
