commit f22154ab465a7778bb9083f48902d3dbde7180a9 (HEAD -> dont-hardcode-swap)
Author: Emil Renner Berthing <systemd@esmil.dk>
Date:   Sun Aug 14 12:47:05 2016 +0200

    units: don't hardcode swap.target

diff --git a/Makefile.am b/Makefile.am
index 0c27f81..f6a63bb 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -267,7 +267,7 @@ LOCAL_FS_TARGET_WANTS =
 MULTI_USER_TARGET_WANTS =
 GRAPHICAL_TARGET_WANTS =
 RESCUE_TARGET_WANTS =
-SYSINIT_TARGET_WANTS =
+SYSINIT_TARGET_WANTS = swap.target
 SOCKETS_TARGET_WANTS =
 BUSNAMES_TARGET_WANTS =
 TIMERS_TARGET_WANTS =
diff --git a/units/sysinit.target b/units/sysinit.target
index ec33503..a7de50a 100644
--- a/units/sysinit.target
+++ b/units/sysinit.target
@@ -9,5 +9,5 @@
 Description=System Initialization
 Documentation=man:systemd.special(7)
 Conflicts=emergency.service emergency.target
-Wants=local-fs.target swap.target
+Wants=local-fs.target
 After=local-fs.target swap.target emergency.service emergency.target
