commit 1887be55401c10e83a6e240473ccb99c863c78fe (HEAD -> ldconfig)
Author: Emil Renner Berthing <systemd@esmil.dk>
Date:   Sun Aug 14 18:46:05 2016 +0200

    units/ldconfig.service: fix for Rodent Linux

diff --git a/units/ldconfig.service b/units/ldconfig.service
index d7b78ba..eb71a08 100644
--- a/units/ldconfig.service
+++ b/units/ldconfig.service
@@ -12,10 +12,8 @@ DefaultDependencies=no
 Conflicts=shutdown.target
 After=local-fs.target
 Before=sysinit.target shutdown.target systemd-update-done.service
-ConditionNeedsUpdate=|/etc
-ConditionFileNotEmpty=|!/etc/ld.so.cache
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
-ExecStart=/sbin/ldconfig -X
+ExecStart=/usr/bin/ldconfig -X
