commit b7ffdc07c515a2e630c380f6db92c4c23836e7c4 (HEAD -> no-sbin-path)
Author: Emil Renner Berthing <systemd@esmil.dk>
Date:   Sun Aug 14 12:48:55 2016 +0200

    Don't include /usr/sbin in default path

diff --git a/src/basic/path-util.h b/src/basic/path-util.h
index a27c13f..82a98a8 100644
--- a/src/basic/path-util.h
+++ b/src/basic/path-util.h
@@ -26,7 +26,7 @@
 #include "macro.h"
 #include "time-util.h"
 
-#define DEFAULT_PATH_NORMAL "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin"
+#define DEFAULT_PATH_NORMAL "/usr/local/sbin:/usr/local/bin:/usr/bin"
 #define DEFAULT_PATH_SPLIT_USR DEFAULT_PATH_NORMAL ":/sbin:/bin"
 
 #ifdef HAVE_SPLIT_USR
