diff -Naur systemd-231.orig/src/systemctl/systemctl.c systemd-231/src/systemctl/systemctl.c
--- systemd-231.orig/src/systemctl/systemctl.c	2016-07-25 21:49:47.000000000 +0200
+++ systemd-231/src/systemctl/systemctl.c	2016-08-13 22:55:05.865752620 +0200
@@ -5356,17 +5356,6 @@
         }
 
         init = empty_to_null(init);
-        if (init) {
-                const char *root_systemd_path = NULL, *root_init_path = NULL;
-
-                root_systemd_path = strjoina(root, "/" SYSTEMD_BINARY_PATH);
-                root_init_path = strjoina(root, "/", init);
-
-                /* If the passed init is actually the same as the
-                 * systemd binary, then let's suppress it. */
-                if (files_same(root_init_path, root_systemd_path) > 0)
-                        init = NULL;
-        }
 
         r = acquire_bus(BUS_MANAGER, &bus);
         if (r < 0)
