commit 66931df2ff5aa4d461199760ffb57466e8cb7a8a (HEAD -> no-stack-protection)
Author: Emil Renner Berthing <systemd@esmil.dk>
Date:   Sun Aug 14 12:51:36 2016 +0200

    Don't use -fstack-proctor and friends

diff --git a/configure.ac b/configure.ac
index cf595e6..8f4b013 100644
--- a/configure.ac
+++ b/configure.ac
@@ -189,10 +189,7 @@ CC_CHECK_FLAGS_APPEND([with_cflags], [CFLAGS], [\
         -fdiagnostics-show-option \
         -fno-strict-aliasing \
         -fvisibility=hidden \
-        -fstack-protector \
-        -fstack-protector-strong \
-        -fPIE \
-        --param=ssp-buffer-size=4])
+        -fPIE])
 
 CC_CHECK_FLAG_APPEND([with_cflags], [CFLAGS], [-Werror=shadow], [
 #include <time.h>
@@ -217,10 +214,6 @@ AS_CASE([$CFLAGS], [*-O[[12345sz\ ]]*],
         [AC_MSG_RESULT([skipping -flto, optimization not enabled])])
 AC_SUBST([OUR_CFLAGS], "$with_cflags $sanitizer_cflags")
 
-AS_CASE([$CFLAGS], [*-O[[12345sz\ ]]*],
-        [CC_CHECK_FLAGS_APPEND([with_cppflags], [CPPFLAGS], [\
-               -Wp,-D_FORTIFY_SOURCE=2])],
-        [AC_MSG_RESULT([skipping -D_FORTIFY_SOURCE, optimization not enabled])])
 AC_SUBST([OUR_CPPFLAGS], "$with_cppflags $sanitizer_cppflags")
 
 AS_CASE([$CFLAGS], [*-O[[12345sz\ ]]*],
