diff -Naur systemd-227.orig/configure.ac systemd-227/configure.ac
--- systemd-227.orig/configure.ac	2015-10-07 17:06:22.000000000 +0200
+++ systemd-227/configure.ac	2015-10-12 18:30:12.565241195 +0200
@@ -190,10 +190,7 @@
         -fdiagnostics-show-option \
         -fno-strict-aliasing \
         -fvisibility=hidden \
-        -fstack-protector \
-        -fstack-protector-strong \
-        -fPIE \
-        --param=ssp-buffer-size=4])
+        -fPIE])
 
 CC_CHECK_FLAG_APPEND([with_cflags], [CFLAGS], [-Werror=shadow], [
 #include <time.h>
@@ -218,10 +215,6 @@
         [AC_MSG_RESULT([skipping -flto, optimization not enabled])])
 AC_SUBST([OUR_CFLAGS], "$with_cflags $sanitizer_cflags")
 
-AS_CASE([$CFLAGS], [*-O[[12345sz\ ]]*],
-        [CC_CHECK_FLAGS_APPEND([with_cppflags], [CPPFLAGS], [\
-               -Wp,-D_FORTIFY_SOURCE=2])],
-        [AC_MSG_RESULT([skipping -D_FORTIFY_SOURCE, optimization not enabled])])
 AC_SUBST([OUR_CPPFLAGS], "$with_cppflags $sanitizer_cppflags")
 
 AS_CASE([$CFLAGS], [*-O[[12345sz\ ]]*],
